modules/apps/foundation/frontend-js/frontend-js-loader-modules-extender/src/main/java/com/liferay/frontend/js/loader/modules/extender/JSLoaderModulesServlet.java

```diff
@@ -97,7 +95,6 @@ protected void service(
 			printWriter.write('@');
 			printWriter.write(jsLoaderModule.getVersion());
 			printWriter.write("': '");
-			printWriter.write(PortalUtil.getPathContext());
 			printWriter.write(jsLoaderModule.getContextPath());
```

Comment:
So the problem should be another completely different, because the `jsLoaderModule.getContextPath()` is already retrieving the context path of the web application, but without the ProxyPath.  So I guess adding same code as in https://github.com/brianchandotcom/liferay-portal/pull/38527/files#diff-837e8d6de3dc9bc0cc14a3cbd6e6d80aR188 will include both